src/dashboard/Dashboard.js

```diff
@@ -226,7 +226,7 @@ class Dashboard extends React.Component {
           <Route path='config' component={Config} />
           <Route path='api_console' component={ApiConsole} />
           <Route path='migration' component={Migration} />
-          <Redirect from='push' to='/apps/:appId/push/activity/all' />
+          <Redirect from='push' to='/apps/:appId/push/new' />
```

Comment:
are we not planning to have the index page? Why not have empty index page for now and start creation flow with create new?